core/src/main/java/org/javarosa/xform/parse/XFormParserFactory.java

```diff
@@ -27,32 +27,11 @@ public XFormParserFactory(Interner<String> stringCache) {
 
     public XFormParser getXFormParser(Reader reader) {
         XFormParser parser = new XFormParser(reader);
-        init(parser);
-        return parser;
-    }
 
-    private void init(XFormParser parser) {
```

Comment:
are we sure none of this affected J2ME stuff, out of curiosity? Is there a way to run a quick Jenkins test or something?